_shows/16_17/escape_for_dummies_edin.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Escape for Dummies"
+devised: true 
+period: Edinburgh
+season: Edinburgh
+season_sort: 590
+date_start: 2017-08-05
+date_end: 2017-08-25
+venue: Royal Terrace, Greenside
+venue_sort: Greenside 
+
+assets:
+  - type: poster
+    image: NfnQCWk
+
+links:
+  - type: Review
+    href: http://edinburgh.fringeguru.com/reviews/edinburgh-2017/escape-for-dummies
+    snapshot: Ju6HL
+    publisher: FringeGuru
+    author: Richard Stamp
+    title: "Escape for Dummies"
+    date: 2017-08-19
+    stars: 5
+    quote: "Escape For Dummies is an hour of pure joy – hilarious and giddy, yet with characters who make you care. Plan ahead for this one, because it's performing on odd-numbered dates only… and you'd have to be a dummy to miss it."
```

Comment:
Would shorten the end to:
`characters who make you care. Plan ahead for this one [...] you'd have to be a dummy to miss it.`